test_plans/pointer_address_space_deduction.asciidoc

```diff
@@ -0,0 +1,49 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan of address space deduction for C++ pointer type variables in kernel function
```

Comment:
My personal belief is that this area of the spec is too poorly defined to write CTS tests.  The tests proposed in this plan are extremely simple, but even these simple tests do not seem to be guaranteed by the spec.  For example, the section "Checking the address space deducing for function return value of pointer type" defines a function that does pointer arithmetic on a function parameter.  I don't see anything in [section 5.9.4](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#subsec:inferredAddressSpace) that guarantees inference rules for pointer arithmetic.

My guidance is to wait for a future version of the spec to clarify this area before writing CTS tests.

Reply:
Ok, got it, thank you. I'll close the PR for now.